plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -1195,6 +1195,32 @@ items:
     - create
     - patch
     - update
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
+    creationTimestamp: null
+    labels:
+      kubernetes.io/bootstrapping: rbac-defaults
+    name: system:controller:root-ca-cert-publisher
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    verbs:
+    - create
+    - update
+  - apiGroups:
+    - ""
+    - events.k8s.io
+    resources:
+    - events
+    verbs:
+    - create
+    - patch
```

Comment:
does rootcacertpublisher need "patch" for events?

Reply:
yes, that's part of the standard event permissions used by the event recorder:

https://github.com/kubernetes/kubernetes/blob/71fea80155e5c124b1a44315da59af9799e9fafc/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L53-L55